_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -15,8 +15,37 @@ The following tools are required:
 * [Terraform](https://www.terraform.io/downloads.html) - 0.12.x
 * [gsutil](https://cloud.google.com/storage/docs/gsutil)
 
+## Service Account Credentials
+Terraform uses an IAM Service Account to deploy and configure resources on behalf of the user.  This
+service account and key can be created by executing the [setup.sh](https://github.com/forseti-security/terraform-google-forseti/blob/master/helpers/setup.sh)
+helper script.
+
+### Obtain the Setup Script
+The following will download the setup script locally.
+```sh
+curl --location --remote-name https://raw.githubusercontent.com/forseti-security/terraform-google-forseti/master/helpers/setup.sh
```

Comment:
Food for thought. We have these same instructions on the TF repo and soon to be on the install page of the website. Do we need to duplicate these steps here?

Reply:
@gkowalski-google Will the instructions in the the install page of the website be phrased in such a way that they these steps are for both GKE and GCE.  Historically, the install page has just been for GCE.  To be clear, I have no issue having "common" instructions in a "common" place.

Reply:
Regarding cloning the repo, I'll make it more generic.  How about:
```
Obtain the [setup.sh](https://github.com/forseti-security/terraform-google-forseti/blob/module-release-5.0.0/helpers/setup.sh) helper script from the terraform-google-forseti repo.
```

Reply:
I think the changes @hshin-g can be updated to support both. Perhaps we can instruct users to run through the setup script, and then either continue reading on to deploy GCE or _click here_ to deploy GKE.